src/helper-functions.ts

```diff
@@ -0,0 +1,65 @@
+Date.prototype.toISOLocaleString = function (): string {
+    return `${this.getFullYear()}-${(this.getMonth()+1).leftPad(2)}-${this.getDate().leftPad(2)}T${this.getHours().leftPad(2)}:${this.getMinutes().leftPad(2)}`;
+}
```

Comment:
not sure if this came along with the task itself, but attaching new functions to prototype is not recommended as we as developers share the same context, so it would be practically impossible to track down who added what, since it can also happen during runtime and may break others code. Usually such problems come from using pollyfills - ideally only infra would define such cases.

Reply:
Here it could be a simple exported function